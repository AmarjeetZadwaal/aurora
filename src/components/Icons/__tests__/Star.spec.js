import React from "react";
import renderer from "react-test-renderer";

import { StarIcon } from "../";

describe("StarIcon", () => {
  it("renders correctly", () => {
    const component = renderer.create(<StarIcon size={22} color="#000" />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
