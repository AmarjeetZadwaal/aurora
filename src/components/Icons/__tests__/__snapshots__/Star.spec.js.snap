// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarIcon renders correctly 1`] = `
<svg
  fill="#000"
  height={22}
  viewBox="0 0 11 10"
  width={22}
>
  <title>
    Star Icon
  </title>
  <path
    d="M5.5 7.7L2.267 9.95l1.14-3.77L.27 3.8l3.938-.08L5.5 0l1.293 3.72 3.938.08-3.139 2.38 1.14 3.77z"
    fillRule="evenodd"
  />
</svg>
`;
